### PR TITLE
CNV-19842: adding storage charts metrics tab

### PR DESCRIPTION
**Pull request by metalice**
## 📝 Description

Add storage total bandwidth chart and storage IOPS total charts to VM metrics tab

## 🎥 Demo

image

### CODE REVIEW

**Comment by metalice** [comment]
/hold

waiting for this to merge first: https://github.com/kubevirt-ui/kubevirt-plugin/pull/796

File: src/utils/components/Charts/StorageUtil/StorageIOPSTotalThresholdChart.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        <Link to={queriesToLink(queries?.STORAGE_IOPS_TOTAL)}>
+          <Chart
+            height={height}
+            width={width}
+            padding={35}
+            scale={{ x: 'time', y: 'linear' }}
+            containerComponent={
+              <ChartVoronoiContainer
+                labels={({ datum }) => {
+                  return `IOPS Total: ${datum?.y?.toFixed(2)}`;
```
Comment (avivtur) [inline src/utils/components/Charts/StorageUtil/StorageIOPSTotalThresholdChart.tsx:59]: this doesn't needs translation? and can you please change remove the return statement?

File: src/utils/components/Charts/StorageUtil/StorageTotalReadWriteThresholdChart.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        <Link to={queriesToLink(queries?.FILESYSTEM_TOTAL_USAGE)}>
+          <Chart
+            height={height}
+            width={width}
+            padding={35}
+            scale={{ x: 'time', y: 'linear' }}
+            containerComponent={
+              <ChartVoronoiContainer
+                labels={({ datum }) => {
+                  return `Data transfer: ${xbytes(datum?.y, { iec: true, fixed: 2 })}`;
```
Comment (avivtur) [inline src/utils/components/Charts/StorageUtil/StorageTotalReadWriteThresholdChart.tsx:63]: same here

**Pull request merged**